src/assets/css/site.scss

```diff
@@ -81,7 +81,8 @@ div[data-role="page"] {
 .content-primary,
 .padded-bottom-page,
 .page,
-.pageWithAbsoluteTabs .pageTabContent {
+.pageWithAbsoluteTabs .pageTabContent,
+.itemsContainer {
```

Comment:
`itemContainer` is too generic - all containers obtain bottom padding. Instead, you can add `padded-bottom-page` to `detailPageWrapperContainer` on music item details page.

Reply:
Yeah, I agree. I amended as you said, and it seems to work.